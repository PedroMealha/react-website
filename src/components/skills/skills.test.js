import { shallow } from "enzyme";
import Skills from "./skills.component";

describe('Skill component', () => {

	const mockStack = {
		coding: [
			{
				0: {
					learning: false,
					percent: 95,
					title: 'HTML'
				}
			}
		]
	}

	const wrapper = shallow(<Skills stackMap={ mockStack } />)

	it('expect to render', () => {
		expect(wrapper).toMatchSnapshot();
	})
});
