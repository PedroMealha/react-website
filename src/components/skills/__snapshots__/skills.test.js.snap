// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skill component expect to render 1`] = `
<Fragment>
  <h4>
    coding
  </h4>
  <div
    className="skills"
  >
    <Progress
      skill={
        Object {
          "0": Object {
            "learning": false,
            "percent": 95,
            "title": "HTML",
          },
        }
      }
    />
  </div>
  <p
    className="learning"
  >
    <span>
      *
    </span>
     actively learning
  </p>
</Fragment>
`;
